tests/conftest.py

```diff
@@ -89,7 +91,15 @@ def gen_dummy_object(class_title, doc):
                 else:
                     type_ = prop.kwargs.get('range')
                     if type_ is not None:
-                        object_[prop.title] = random.randint(50,100)
+                        if "dateTime" in type_:
+                            value = datetime.utcnow()
+                            obj = datetime.strftime(value, "%d/%m/%Y %H:%M:%S")
```

Comment:
same as above. you should avoid using hard-coded arbitrary formats for date-times, we need to stick to conventiion that in the datetime case is the [ISO 8601 format](https://www.cl.cam.ac.uk/~mgk25/iso-time.html).
check [this post](https://stackoverflow.com/questions/127803/how-do-i-parse-an-iso-8601-formatted-date) about the timezone information: timezone in the database has to be always UTC, the client should pass the datetime always with the timezone specification (so it has to be checked by the server). The ideal ISO 8601 format in my experience is `2008-09-03T20:56:35.450686Z`, see the `T` separator between date and time and the indication of the timezone `Z`